ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/tables/management-security.md

```diff
@@ -116,4 +124,28 @@
           # If specified, one CRL needs to be provided for every certificate in the chain, even if the revocation list in the CRL is empty.
           certificate_revocation_lists:
             - <str>
+      shared_secret_profiles:
+        - profile_name: <str; required; unique>
+          secrets:
+            - name: <str>
+              secret: <str>
+
+              # Specify the lifetime of password. Below options are available. This value is applicable to receive-lifetime and transmit lifetime.
+              # If lifetime, receive-lifetime and transmit-lifetime are specified then AVD will apply the value of lifetime to both receive and transmit lifetime.
+              # infinite    -    infinite lifetime
+              # mm/dd/yyyy  -    date
+              # yyyy-mm-dd  -    date
+              lifetime: <str>
```

Comment:
it has few more option, will you please take look.
```
wan1-dc1(config-mgmt-sec-sh-sec-profile-tes)#secret te te1 infinite ?
  local-time  Configuring secrets using the local timezone from system clock. Default is UTC
  <cr>        

wan1-dc1(config-mgmt-sec-sh-sec-profile-tes)#secret te te1 infinite local-time ?
  <cr>  

wan1-dc1(config-mgmt-sec-sh-sec-profile-tes)#secret te te1 10/01/2024 ?
  hh:mm:ss  Time in hh:mm:ss format

wan1-dc1(config-mgmt-sec-sh-sec-profile-tes)#secret te te1 10/01/2024 12:00:00 ?
  infinite    infinite lifetime
  mm/dd/yyyy  Date
  yyyy-mm-dd  Date

wan1-dc1(config-mgmt-sec-sh-sec-profile-tes)#secret te te1 10/01/2024 12:00:00 11/11/2025 ?
  hh:mm:ss  Time in hh:mm:ss format

wan1-dc1(config-mgmt-sec-sh-sec-profile-tes)#secret te te1 10/01/2024 12:00:00 11/11/2025 12:00:00 ?
  local-time  Configuring secrets using the local timezone from system clock. Default is UTC
  <cr>        

wan1-dc1(config-mgmt-sec-sh-sec-profile-tes)#secret te te1 10/01/2024 12:00:00 11/11/2025 12:00:00 local-time ?
  <cr>  
```

Let me know if you see any difference in your EOS version, will help you to get other options.

Reply:
Fixed.